docs/articles/blog/2020-7-15-introducing-the-testcafe-jenkins-plugin.md

```diff
@@ -0,0 +1,170 @@
+
+Install the main testcafe library, as well as the [testcafe-reporter-jenkins](https://www.npmjs.com/package/testcafe-reporter-jenkins) package. The latter is needed to generate Jenkins-friendly JUnit format reports.
+
+To do this, add the following *build step* to your Jenkins project:
+
+```bash
+npm i testcafe testcafe-reporter-jenkins
+```
+
+### Step 4. Configure the tests
+
+You can configure your tests via:
+
+* the [command line interface](https://devexpress.github.io/testcafe/documentation/reference/command-line-interface.html)
+* the JavaScript/TypeScript API
+* the .testcaferc.json [configuration file](https://devexpress.github.io/testcafe/documentation/reference/configuration-file.html)
+
+The last option provides an easy, declarative way to define your test settings. You can store this file in the same folder as your tests.  Below is the configuration we're going to use in this tutorial:
+
+```js
+{
+    "browsers": [
+        "chromium:headless:emulation:device=iPhone 4",
+        "chromium:headless:emulation:device=iPad Mini",
+        "chromium:headless"
+    ],
+    "src": "test/e2e/**/*",
+    "screenshots": {
+        "takeOnFails": true,
+        "fullPage": true
+    },
+    "videoPath": "videos",
+    "reporter": {
+        "name": "jenkins",
+        "output": "report.xml"
+    },
+    "appCommand": "node server.js"
+}
+```
+
+The *Browser* array contains the list of browsers TestCafe will use to run our test. The [TestCafe browser syntax](https://devexpress.github.io/testcafe/documentation/guides/concepts/browsers.html#use-chromium-device-emulation) allows us to specify the mobile devices we want Chromium to emulate. 
+
+The *name* property of the *reporter* object is set as *jenkins*, which ensures that the reports generated by TestCafe can be properly parsed by the server.
+
+If you decide against using a config file, don't forget to [manually set](https://devexpress.github.io/testcafe/documentation/reference/command-line-interface.html#-r-nameoutput---reporter-nameoutput) the report format when launching your tests, like so:
+
+```bash
+testcafe chrome test.js -r jenkins:report.xml
+```
+
+Otherwise, just run the following command:
+
+```bash
+npx testcafe
+```
+
+### Step 5. Configure the reporter
+
+Turn on the [JUnit](https://plugins.jenkins.io/junit/) reporter plugin: add the "Publish JUnit test result report" *post-build action*. To display screenshots and videos alongside your test results, select the option "Include links to TestCafe artifacts" from the "Additional test report features" drop-down.
+
+Important: Don't forget to check the "Retain long standard output/error" box. Otherwise the plugin will automatically truncate information concerning test screenshots and videos.
+
+![JUnit reporter configuration screen](/testcafe/images/jenkins-plugin/junit-reporter-configuration.png)
+
+### Step 6. Run the build
+
+Save the changes to your project and press the "Build now" button on the project page:
```

Comment:
Do you say "press" because you are talking about a mobile device?

Reply:
Thanks for pointing this out! I was under the impression that the verb _to press_ is used to describe interactions with UI elements that enter a visually distinct 'pressed' state when clicked. Upon consulting two different style guides, I realized that I was mistaken, and that I should only really use the word 'press' when talking about physical I/O devices (i.e. the keyboard or the touch screen). Is that correct?

Reply:
Correct, press should only be used when you are actually "pressing" something, like the Shift key on a keyboard or a button on a tablet.

I think "press" is used in other parts of the article as well. Can you replace them?